docs/main/Solutioning/openebs-on-kubernetes-platforms/EKS/eks.md

```diff
@@ -0,0 +1,547 @@
+
+**Storage Agility:** OpenEBS supports multiple storage engines that can be tailored to your application’s specific needs. This allows you to select the most suitable storage engine for different workloads running on EKS, providing agility in how storage is managed.
+
+**Cloud-Native Best Practices:** Amazon EKS, combined with OpenEBS, allows you to follow cloud-native best practices for deploying, managing, and scaling stateful applications. This approach leverages the benefits of both Kubernetes and cloud infrastructure to build resilient and scalable applications.
+
+## Prerequisites
+
+Before installing Replicated PV Mayastor, make sure that you meet the following requirements:
+
+- **Hardware Requirements**
+
+    Your machine type must meet the requirements defined in the [prerequisites](../../../user-guides/replicated-storage-user-guide/replicated-pv-mayastor/rs-installation.md#prerequisites).
+
+- **EKS Nodes**
+
+    You need to configure launch templates to create node groups with hardware/OS/kernel requirements. When using the synchronous replication feature (N-way mirroring), the number of worker nodes on which IO engine pods are deployed should be no less than the desired replication factor. 
+
+- **Additional Disks**
+
+    Additional storage disks for nodes can be added during the cluster creation using launch templates.
+
+- **Ports**
+
+    Ensure security groups are having the OpenEBS ports allowed. Refer to the [Replicated PV Mayastor Installation Documentation](../../../user-guides/replicated-storage-user-guide/replicated-pv-mayastor/rs-installation.md#network-requirements) for more details.
+
+- **Enable Huge Pages**
+    
+    2MiB-sized Huge Pages must be supported and enabled on the storage nodes i.e. nodes where IO engine pods are deployed. A minimum number of 1024 such pages (i.e. 2GiB total) must be available exclusively to the IO engine pod on each node.
+    Secure Socket Shell (SSH) to the EKS worker node to enable huge pages.
+
+- **Kernel Modules**
+
+    SSH to the EKS worker nodes to load nvme_tcp modules.
+
+    ```
+    modprobe nvme_tcp
+    ```
+
+- **Preparing the Cluster**
+
+    Refer to the [Replicated PV Mayastor Installation Documentation](../../../user-guides/replicated-storage-user-guide/replicated-pv-mayastor/rs-installation.md#preparing-the-cluster) for instructions on preparing the cluster.
+
+## Install Replicated PV Mayastor on Amazon EKS
+
+Refer to the [OpenEBS Installation Documentation](../../../quickstart-guide/installation.md#installation-via-helm) to install Replicated PV Mayastor using Helm.
+
+- **Helm Install Command**
+
+```
+helm install openebs --namespace openebs openebs/openebs --create-namespace
+```
+
+## Pools
+
+The available local SSD disks on worker nodes can be viewed by using the `kubectl-mayastor` plugin.
```

Comment:
Is this statement correct? I think this was for instance volumes.

Reply:
It's possible to view for both instances store/local ssd volumes and normal instances.